storages/redis/redis.go

```diff
@@ -189,5 +188,5 @@ func (s *Storage) decode(data []byte, to interface{}) error {
 }
 
 func wrapError(err error, msg string) error {
-	return errors.Wrapf(err, "fsm-telebot/storage/redis: %s", msg)
+	return fmt.Errorf("fsm-telebot/storage/redis: %w: %s", err, msg)
```

Comment:
The message must be before the error. Because it is used to describe the operation.

```suggestion
	return fmt.Errorf("fsm-telebot/storage/redis: %s: %w", msg, err)
```